Final status:
Process still alive after running test suite
stdout and stderr:
DEBUG:root:[*] running method core_negotiate_tlv_encryption
DEBUG:root:[*] Negotiating TLV encryption
DEBUG:root:[*] RSA key: 30820122300d06092a864886f70d01010105000382010f003082010a0282010100c546604a125306453e6aaf8e1170a411590b20ec30dea15341192b531697ea5966898b6f1823730c5780a1704416cc5a20ee2fae5bf6aada6c4ca38a41cfa91bd0b95eab5a50544a4ed197ac16a440b91c587cc30ab58e5325e75f37a64fd70fb7f3e106befb8f2686cb4f998210ac139286920f944279c92fd1068c6953580d15f30b6dd71dd1bb64e3a6b99f5912cdb43b99540f76685f5efdb436819c7766414078bb5f54da37405c27b5dfd4d133c8db8c66e68ea715dbe328ed686dc8aa2b3db84217a2d506409e641103074e309c6a5f0fffe243b37225ab5d3439b4d92da1d1ef6b3a5ffd101e84616a6c9b01f103e11e00cb3af9ee16f667fe93808b0203010001
DEBUG:root:[*] AES key: 0xc873784b6fdd1458b283c385c8fcc316244878f73dac2d380e3a891c289883f0
DEBUG:root:[*] Encrypted AES key: 0x6d22033ebacd2fd49808d033b1795604e98d6936252b55b321b5d785d0e3f66fff4af296f0f5fc4f4e79bf8950cafd93c225e01c8f046dda2475ada6cd532bf3e148bb97594abea129b6243da19380194a21b477f38640d9c63561134db206e487f0172803c527aa50423e594ed8679ba338b1fb19c21b1fd0a982b612608baa357ba39248ba587a9f4caa2994b2281985a26e9574560c0a1be6a4d300cd5fc635cb0521cc77fa51228b7c9501326c79209b695c4adf35555daae2b2a513ea5f158b8e01bf5c2930330d9dc2b2ccd3d0379d26a51f97d924f2ce84f8470dc38eaf8295b64b27f5ae39babf623ef944efc5d060ce6909ca1a5edfe3910a4985cf
DEBUG:root:[*] TLV encryption sorted
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_set_session_guid
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_enumextcmd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_enumextcmd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_loadlib
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getuid
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_sysinfo
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_set_uuid
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_net_config_get_interfaces
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_net_config_get_routes
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_separator
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_chdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir-zormapey'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_chdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 1 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 1
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 2 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 2
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_file
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 3 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 3
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 4 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 4
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 5 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 5
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_file
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\boot.ini'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\pagefile.sys'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: '/etc/master.passwd'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: '%WINDIR%\\system32\\notepad.exe'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: '%WINDIR%\\system32\\calc.exe'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 6 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 6
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 7 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 7
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 8 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 8
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 9 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 9
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 10 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 10
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 11 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 11
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_file
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-moved'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 12 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 12
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-moved'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_file_move
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_file
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 13 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 13
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir/directory'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 14 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 14
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_ls
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_dir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_execute
DEBUG:root:[*] starting process: ['/bin/sh', '-c', 'command -v ln || which ln && echo LGGPMGBA']
DEBUG:root:[*] added process id: 3527, handle: 1
DEBUG:root:[*] added channel id: 15 type: MeterpreterProcess
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[-] method core_channel_read resulted in error: #1
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_close
DEBUG:root:[*] closed and removed process handle: 1
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 16 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 16
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_execute
DEBUG:root:[*] starting process: ['/bin/sh', '-c', 'ln -s $(pwd)/meterpreter-test-dir $(pwd)/meterpreter-test-dir.lnk']
DEBUG:root:[*] added process id: 3531, handle: 2
DEBUG:root:[*] added channel id: 17 type: MeterpreterProcess
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[-] method core_channel_read resulted in error: #1
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_close
DEBUG:root:[*] closed and removed process handle: 2
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_dir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.lnk'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_dir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.1'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.2'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 18 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 18
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_execute
DEBUG:root:[*] starting process: ['/bin/sh', '-c', 'ln -s $(pwd)/meterpreter-test-dir.1 $(pwd)/meterpreter-test-dir.2/link']
DEBUG:root:[*] added process id: 3537, handle: 3
DEBUG:root:[*] added channel id: 19 type: MeterpreterProcess
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_close
DEBUG:root:[*] closed and removed process handle: 3
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_dir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_dir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_chdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
